website/static/css/donate-banner.css

```diff
@@ -0,0 +1,38 @@
+/*Background photo © Nestlé*/
```

Comment:
While it is CC, they ask that you attribute them https://www.flickr.com/photos/nestle/8144369359/in/photolist-dpG2gn-dpG3x5-dpG3pu-dpFSfg-dpG2wF-ieiWeh-7n6syc-dpG1RD-TviA5S-Tyds7i-UDLmRo-W39f4J-igSsqU-VdCDox-RnMtkU-PH8Pi-5t5QGk-Ejp7Q-9NMWTu-27NH6k-hFh5mM-535xiH-fKmgSg-539Npd-9FkJxj-97vLHq-E8NCL-pzgPcA-77ZWJZ-6qiS87-mVpeki-dWEV1u-e54fGC-Fdu2uf-RNHsa7-UJk4Go-RDSvmS-5oo9Wt-SZkr5m-Q1nNvp-V2Nm1o-WCWwBw-VRXJpS-PrAjCY-SLELS1-Q1o4QT-W2kWcH-PrAkAu-NJ523N-PrAgDS